hist/hist/src/TGraph.cxx

```diff
@@ -2008,6 +2011,70 @@ Int_t TGraph::RemovePoint(Int_t ipoint)
    return ipoint;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Save the graph as .csv, .tsv or .txt. In case of any other extension, fall
+/// back to TObject::SaveAs
+///
+/// The result can be immediately imported into Excel, gnuplot, Python or whatever,
+/// without the needing to install pyroot, etc.
+///
+/// \param filename the name of the file where to store the graph
+/// \param option some tuning options
+///
+/// The file extension defines the delimiter used:
+///  - `.csv` : comma
+///  - `.tsv` : tab
+///  - `.txt` : space
+///
+/// If option = "title" a title line is generated with the axis titles.
+
+void TGraph::SaveAs(const char *filename, Option_t *option) const
+{
+   TString del = "";
```

Comment:
maybe const char del = '\0' is simpler than a TString? (so that you do not need del.Data() later on in several places, just "del").